src/module-elasticsuite-tracker/Model/Customer/TrackingService.php

```diff
@@ -142,6 +142,7 @@ private function addCustomerLink($eventData)
 
                 $this->customerLinkResource->saveLink($data);
             }
+            unset($eventData['customer']['id']); // Do not persist the customer_id in ES index to preserve anonymization.
```

Comment:
Hmmm, unsetting something in $eventData in addCustomerLink will hardly have any effect to the event written to database beforehand on line 79, won't it ?
If we don't want to commit the customer id to the ES index, should we re-implement the whole "addCustomerLink" as eventProcessor ?
Or at least extract in a local variable the customerId, remove it from eventData and pass both to addCustomerLink ?
Or again call addCustomerLink *before* eventQueue->addEvent ?

Reply:
yes, lines 79 and 80 should be substituted